dask_expr/io/parquet.py

```diff
@@ -330,6 +330,115 @@ def to_parquet(
     engine=None,
     **kwargs,
 ):
+    """Store Dask.dataframe to Parquet files
+
+    Notes
+    -----
+    Each partition will be written to a separate file.
+
+    Parameters
+    ----------
+    df : dask.dataframe.DataFrame
+    path : string or pathlib.Path
+        Destination directory for data.  Prepend with protocol like ``s3://``
+        or ``hdfs://`` for remote data.
+    compression : string or dict, default 'snappy'
+        Either a string like ``"snappy"`` or a dictionary mapping column names
+        to compressors like ``{"name": "gzip", "values": "snappy"}``. Defaults
+        to ``"snappy"``.
+    write_index : boolean, default True
+        Whether or not to write the index. Defaults to True.
+    append : bool, default False
+        If False (default), construct data-set from scratch. If True, add new
+        row-group(s) to an existing data-set. In the latter case, the data-set
+        must exist, and the schema must match the input data.
+    overwrite : bool, default False
+        Whether or not to remove the contents of `path` before writing the dataset.
+        The default is False.  If True, the specified path must correspond to
+        a directory (but not the current working directory).  This option cannot
+        be set to True if `append=True`.
+        NOTE: `overwrite=True` will remove the original data even if the current
+        write operation fails.  Use at your own risk.
```

Comment:
Out of scope: At first glance, these two options appear to be the inverse of one another. That is not the case though. We may want to do a better job at explaining this. (I still don't understand it.)

Reply:
Yeah the parquet docs are ridiculously bad at actually explaining at what every option does

Reply:
I addded https://github.com/dask/dask-expr/issues/1054 as a reminder.